modyn/models/dlrm/cuda_ext/sparse_embedding.py

```diff
@@ -57,7 +58,7 @@ class JointSparseEmbedding(nn.Module):
         device (torch.device): where to create the embedding. Default "cuda"
     """
 
-    def __init__(self, categorical_feature_sizes, embedding_dim, device="cuda"):
+    def __init__(self, categorical_feature_sizes: Sequence[int], embedding_dim, device="cuda"):
```

Comment:
embedding dim and device should maybe also get typing, while we are at it

Reply:
Ah sure, didn't notice :) we can also force argument typing in the future :) I can do that when setting up a mypy config.